### ISSUE TITLE
can't evaluate field ID in type generator.Options

### ISSUE THREAD

**Issue by leoujz** [go-micro/cli#15]
go version go1.18.7 linux/arm64
go-micro version v1.1.3

Run
```
go-micro new service --jaeger helloworld
```

Result
```
creating service helloworld
template: helloworld/main.go:47:7: executing "helloworld/main.go" at <.ID>: can't evaluate field ID in type generator.Options
```

then the generation is interrupted

**Comment by Davincible** [comment]
This was fixed in the latest v1.1.4 :)

**Comment by leoujz** [comment]
thanks, 

> This was fixed in the latest v1.1.4 :)

solved, thank you. I think the version in README might need be updated

**Comment by Davincible** [comment]
Good point, thank you
